lib/sql_tracker/handler.rb

```diff
@@ -62,12 +62,14 @@ def trace_path_matcher
 
     def clean_sql_query(query)
       query.squish!
-      query.gsub!(/(\s(=|>|<|>=|<=|<>|!=)\s)('[^']+'|[\$\+\-\w\.]+)/, '\1xxx')
-      query.gsub!(/(\sIN\s)\([^\(\)]+\)/i, '\1(xxx)')
-      query.gsub!(/(\sBETWEEN\s)('[^']+'|[\+\-\w\.]+)(\sAND\s)('[^']+'|[\+\-\w\.]+)/i, '\1xxx\3xxx')
-      query.gsub!(/(\sVALUES\s)\(.+\)/i, '\1(xxx)')
-      query.gsub!(/(\s(LIKE|ILIKE|SIMILAR TO|NOT SIMILAR TO)\s)('[^']+')/i, '\1xxx')
-      query.gsub!(/(\s(LIMIT|OFFSET)\s)(\d+)/i, '\1xxx')
+      unless @config&.retain_sql_query_ids
```

Comment:
I think It's better to return the query directly instead of wrapping everything in the conditional block.

So something like:
```suggestion
      return query if @config&.retain_sql_query_ids
```